src/python/CMakeLists.txt

```diff
@@ -43,6 +43,7 @@ endfunction( add_gudhi_debug_info )
 if(PYTHONINTERP_FOUND)
   if(PYBIND11_FOUND)
     add_gudhi_debug_info("Pybind11 version ${PYBIND11_VERSION}")
+    set(GUDHI_PYTHON_MODULES "${GUDHI_PYTHON_MODULES}'random_point_generators', ")
```

Comment:
Most new modules should use the `_EXTRA` version of the macro, which avoids an automatic import.

Reply:
ok, done